chmed23a/terminology.md

```diff
@@ -0,0 +1,904 @@
+| `20033000` | intralymphatische Anwendung | Voie intralymphatique |
+| `20035000` | intramuskuläre Anwendung | Voie intramusculaire |
+| `20036000` | intraokulare Anwendung | Voie intraoculaire |
+| `20036500` | intraossäre Anwendung | Voie intraosseuse |
+| `20038000` | intraperitoneale Anwendung | Voie intrapéritonéale |
+| `20039000` | intrapleurale Anwendung | Voie intrapleurale |
+| `20042000` | intrathekale Anwendung | Voie intrathécale |
+| `20043000` | intratumorale Anwendung | Voie intratumorale |
+| `20044000` | intrauterine Anwendung | Voie intra-utérine |
+| `20045000` | intravenöse Anwendung | Voie intraveineuse |
+| `20046000` | intravesikale Anwendung | Voie intravésicale |
+| `20047000` | intravitreal | Voie intravitréenne |
+| `20049000` | nasale Anwendung | Voie nasale |
+| `20051000` | Anwendung am Auge | Voie ophtalmique |
+| `20053000` | zum Einnehmen | Voie orale |
+| `20054000` | Anwendung in der Mundhöhle | Voie buccale |
+| `20055000` | Anwendung im Mund- und Rachenraum | Voie oropharyngée |
+| `20057000` | periartikuläre Anwendung | Voie périarticulaire |
+| `20058000` | perineurale Anwendung | Voie périneurale |
+| `20059000` | zur periodontalen Anwendung | Voie périodontale |
+| `20059400` | peritumorale Anwendung | Voie péritumorale |
+| `20061000` | rektale Anwendung | Voie rectale |
+| `20061500` | retrobulbäre Anwendung | Voie rétrobulbaire |
+| `20062000` | Art der Anwendung nicht spezifizierbar | Autre(s) |
+| `20065000` | subkonjunktivale Anwendung | Voie sous-conjonctivale |
+| `20066000` | subkutane Anwendung | Voie sous-cutanée |
+| `20067000` | sublingual | Voie sublinguale |
+| `20067500` | submuköse Anwendung | Voie sous-muqueuse |
+| `20070000` | transdermale Anwendung | Voie transdermique |
+| `20071000` | Anwendung in der Harnröhre | Voie urétrale |
+| `20072000` | vaginale Anwendung | Voie vaginale |
+| `20080000` | intracerebroventrikuläre Anwendung | Voie intracérébroventriculaire |
+| `20081000` | subretinale Anwendung | voie subrétinale |
+| `20087000` | extrapleurale Anwendung | Voie extrapleurale |
+
+## Method of Administration
+
+**CodeSystem**: CDTYP62
+
+Code system for method of administration (MOA) codes, 
+based on the method of administration codes of the 'EDQM Standard Terms' codes system, 
+which is accessible over https://standardterms.edqm.eu
+
+| Code | German | French |
+| --- | --- | --- |
+| `4` | Administration | administration |
+| `5` | Applikation | application |
+| `7` | kauen | mâcher |
+| `8` | gurgeln | se gargariser |
+| `9` | Infusion | perfusion |
+| `10` | Inhalation | inhalation |
+| `11` | Injektion | injection |
+| `12` | Insertion | insertion |
+| `13` | Instillation | instillation |
+| `17` | sprühen | pulvériser |
+| `18` | lutschen | sucer |
+| `19` | schlucken | avaler |
+| `112` | Dialyse | dialyse |
+| `113` | Implantation | implantation |
+
+## Unit
+
+**CodeSystem**: CDTYP9
+
+Code system for unit codes
```

Comment:
Where are those accessible? Is it the same as the ones above?

Reply:
AFAIK these are not based on another system but directly defined by HCI.